### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by franzos** [nylas/nylas-mail#3621]

**Comment by moon6pence** [comment]
So sad, it is only mail client in Ubuntu that provides me user experience like in Mac. 

Does anyone know alternative of this project? 
Or, I'd better just keep using latest stable release..

**Comment by bubuzzz** [comment]
This is so sad. The app works great for me and have the decent look on both linux, mac

**Comment by dubstrike** [comment]
Yet they have the decency to charge money for snoozing emails in a dead app.

**Comment by franzos** [comment]
@dubstrike I've been a paying customer for a long time, recommended the app to a lot of people. It's a huge let-down that they don't even have the decency to properly announce the full-stop of development. I surely hope the community will pick-up the pieces and continue development.

Since there are quite a number of paying users, we may as well set-up a fund, to support future development, independent from Nylas. I'm certainly willing to chip-in.

**Comment by bengotow** [comment]
Hey folks! I'm one of the original authors of Nylas Mail (hey, I'm in the screenshot!)

I left Nylas back in December, but I'm also working on a fork of the app over at https://github.com/Foundry376/Mailspring. The first release isn't quite ready, but should be in a couple weeks. I'm bringing up infrastructure to continue to power the cloud features and re-writing the mail sync code (which I was never really satisfied with) in C++ using mailcore2, the same framework used by Airmail and Mailbox.  Stay tuned!

Update: Mailspring is now available at https://getmailspring.com/. Enjoy!

**Comment by darklinki** [comment]
@bengotow  Nice that you keep it up, but I realy dislike that you rewrite it in C++. The best part if nylas was that I can completly configure it by my own with js.

**Comment by corbinu** [comment]
@bengotow Super excited that your keeping it up. I notice that the there is no issue track and the slack invite link does not work. How would I go about getting involved in helping you?

**Comment by diegomachadosoares** [comment]
@bengotow, it's really good that you're keeping it up. I'll try to contribute whenever possible. Two things for now. 1- Does your fork still store user credentials into you servers (that was a serious concern for me)? 2 - I couldn't see any issues area in the fork, will you open it?

**Comment by corbinu** [comment]
@diegomachadosoares  Good point I would be much more comfortable with it storing credentials if the server is open source and I can deploy it myself

**Comment by diegomachadosoares** [comment]
@corbinu The server is actualy open source (see it here https://github.com/nylas/sync-engine), but from what I got by exchanging some emails with the Nylas team, this client stores your credentials into their servers anyway to enable features like snooze and etc... I believe that it should be at least * easily * configurable and at that time it wasn't (at least not without tinkering into the code)... therefore my question.

**Comment by ceilican** [comment]
Is this why snoozed messages stopped coming back to inbox?

**Comment by Bergg** [comment]
@ceilican hah I was wondering about that too..

**Comment by Bergg** [comment]
@moon6pence I'm increasingly using the new thunderbird which can be configured to look a bit like Nylas especially with its new themes http://www.omgubuntu.co.uk/2017/04/a-modern-thunderbird-theme-font but I miss features like snoozing and the way the aliases are handled. One bonus though it does have a spam filter.

**Comment by goldsteal** [comment]
How ironic. I only downloaded Nylas Mail today searching for a proper email client. 
Oh, well... 🗡 R.I.P.
@bengotow : Good luck with the project will follow it closely. :)

**Comment by AyoungDukie** [comment]
Additionally, a group from the community is working on maintaining the existing Nylas codebase (seen here: https://github.com/nylas/nylas-mail/issues/3564

>danielmelogpi
> I gave this a shot and forked the project to https://github.com/nylas-mail-lives/nylas-mail.
>
>I still need to study it in my spare time and I imagine you guys are in the same boat.
>@mayurdw, @sotayamashita and @dweremeichik I sent you invites for membership.
>May be we can try to set this in motion!

Seems to be pretty active, though they are still distributing a dev version only at the moment.

![image](https://user-images.githubusercontent.com/5449443/30246362-a0f50b2e-95c5-11e7-8ce1-c9ddda50610b.png)

**Comment by mikeseese** [comment]
I guess this issue still has some activity. I'm one of the developers at https://github.com/nylas-mail-lives/nylas-mail, and just like to say hi!

A couple of things people may have missed:
- Nylas made their official sunset notice: https://www.nylas.com/blog/sunsetting-nylas-mail-development
- We do have a release (v2.1.0, based off v2.0.15 [which isn't the latest Nylas codebase]) over at https://github.com/nylas-mail-lives/nylas-mail/releases/tag/2.1.0
- We'll be releasing v2.2.0 shortly. The large version jump is because this version will include the entire code dump Nylas made (read the blog post above) and will have all of the latest Nylas-made features that is missing in v2.1.0
- NML has a slack channel; you can get your invite at https://nylasisalive.slack.com/join/shared_invite/MjAzMDE1NTU0MDM2LTE0OTgyNzM3MTItODY4OWNlNTdkMw

Edit: You can find the latest **v2.2.1** build at https://github.com/nylas-mail-lives/nylas-mail/releases/latest; we're releasing v2.2.2 in the next week or so

Also feel free to checkout [Mailspring](https://github.com/Foundry376/Mailspring). Ben Gotow (one of the cofounders of Nylas) has also made a fork. It has a closed-source C++/Obj-C sync engine, and some of the features are premium only.

**Comment by vladshcherbin** [comment]
Wow, such a great product simply died. A really sad story.

**Comment by jrbgit** [comment]
Thank you for posting this! I guess that explains the little bugs I've noticed creeping in.

**Comment by franzos** [comment]
For god's sake someone remove the issues or at least add a comment (as in template) to ensure that people stop posting new issues even though they are never going to get addressed. I can't believe how negligent Nylas is... then again, they abandoned this mail client after making people pay for it.

**Comment by mikeseese** [comment]
@franzos I'll reach out to my connection at Nylas to see if they can lock the Issues down somehow or leave a note in an issue template.
